app/src/main/java/org/oppia/app/help/HelpCategoryAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package org.oppia.app.help
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.RecyclerView
+import org.oppia.app.databinding.HelpItemBinding
+
+/** The Recycler View adapter in the [HelpFragment]. */
+class HelpCategoryAdapter(
+  private val activity: AppCompatActivity,
+  private val arrayList: ArrayList<HelpViewModel>
+) :
+  RecyclerView.Adapter<HelpCategoryAdapter.HelpItemView>() {
+
+  override fun onCreateViewHolder(
+    parent: ViewGroup,
+    viewType: Int
+  ): HelpItemView {
+    val layoutInflater = LayoutInflater.from(parent.context)
+    val helpItemBinding = HelpItemBinding.inflate(
+      layoutInflater,
+      parent,
+      /* attachToParent= */ false
+    )
+    return HelpItemView(helpItemBinding)
+  }
+
+  override fun getItemCount(): Int {
+    return arrayList.size
+  }
+
+  override fun onBindViewHolder(holder: HelpCategoryAdapter.HelpItemView, position: Int) {
```

Comment:
This can be written directly as: `override fun onBindViewHolder(holder: HelpItemView, position: Int) {`